src/SparqlLuaQueryRunner.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\SPARQL;
+
+use MediaWiki\Http\HttpRequestFactory;
+
+class SparqlLuaQueryRunner {
+
+	private HttpRequestFactory $requestFactory;
+	private string $sparqlEndpoint;
+
+	/**
+	 * @param HttpRequestFactory $requestFactory
+	 * @param string $sparqlEndpoint
+	 */
```

Comment:
This docstring is unecessary, since the types are already part of the method signature.